hs_core/models.py

```diff
@@ -718,26 +729,25 @@ def update(cls, element_id, **kwargs):
         metadata_obj = kwargs['content_object']
         # get matching resource
         resource = BaseResource.objects.filter(object_id=metadata_obj.id).first()
-
+        publisher_CUAHSI = "Consortium of Universities for the Advancement of Hydrologic Science, Inc. (CUAHSI)"
```

Comment:
@hyi and @horsburgh Resources for which there are no content files in hydroshare (metadata only resources) the metadata specification says the publisher element should be editable by user. I was thinking there may be a need to update publisher element for these resources even after the resource is published.